README.md

```diff
@@ -49,6 +49,11 @@ logs.)
 
 * `version`: Application version. Included in the `details.log.gz` file.
 
+* `label`: Label to attach to the github issue, and include in the details file.
+
+  If using the JSON upload encoding, this should be encoded as a `labels` field,
```

Comment:
It's bad enough that we have both JSON and multipart support, I'd rather we didn't diverge any more than we have to. Why does the JSON form support multiple labels on the GH issue, but the multipart form is strictly limited to 1 label? Seems bizarre.